OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.Validations.Remove.cs

```diff
@@ -14,6 +15,10 @@ namespace OtripleS.Web.Api.Tests.Unit.Services.Foundations.AssignmentAttachments
 {
     public partial class AssignmentAttachmentServiceTests
     {
+        private object actualAssignmentAttachmentDependencyValidationException;
+        private Guid someGuid;
+        private object expectedAssignmentAttachmenDependencyValidationException;
+
```

Comment:
We don't need these variables here. they should be defined within the test itself.